FileIO/TetGenInterface.cpp

```diff
@@ -230,8 +228,18 @@ MeshLib::Mesh* TetGenInterface::readTetGenMesh (std::string const& nodes_fname,
 		return nullptr;
 	}
 
+	MeshLib::Properties properties;
+	// remove material vector again if all mat-ids are "0"
```

Comment:
???? the material vector is created if any of the materials is not equal 0 ... otherwise (if all mat-ids *are* 0), the mat vec is removed again (or rather not copied into properties in the first place)